### PR TITLE
[KEYCLOAK-7482] Updated description of the brute force algorithm.

### PR DESCRIPTION
**Pull request by douglaspalmer**

### CODE REVIEW

File: server_admin/topics/threat/brute-force.adoc [hunk 1]
```diff
@@ -10,13 +10,39 @@ go to the `Brute Force Detection` sub-tab.
...
-the account is temporarily disabled for the `Wait Increment` multiplied by the number of failures over the max.  After
-`Failure Reset Time` is reached all failures are wiped clean.  The `Max Wait` is the maximum amount of time
-an account can be disabled.  Another preventive measure is that if there are subsequent login failures for one
-account that are too quick for a human to initiate the account will be disabled.  This is controlled by the
-`Quick Login Check Milli Seconds` value.  So, if there are two login failures for the same account within that value,
-the account will be disabled for `Minimum Quick Login Wait`.
+There are 2 different configurations for brute force detection; permanent lockout and temporary lockout. Permanent lockout will disable a users account after an attack is detected; the account will be disabled until an administrator renables it. Temporary lockout will disable a users account for a time period after an attack is detected; the time period for which the account is disabled increases the longer the attack continues. The algorithms for each configuration are described below:
```
Comment (hmlnarik) [inline server_admin/topics/threat/brute-force.adoc]: "The algorithms _and their parameteris_ for each configuration are described below:" should be in separate paragraphs.

File: server_admin/topics/threat/brute-force.adoc [hunk 2]
```diff
@@ -10,13 +10,39 @@ go to the `Brute Force Detection` sub-tab.
...
+
+Permanent Lockout
+Max Login Failures = 30
```
Comment (hmlnarik) [inline server_admin/topics/threat/brute-force.adoc]:
```
Max Login Failures::
  Maximum allowed number of login failures. Default value is 30.
```

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Please fix the formatting and add parameter description

**Review by hmlnarik — CHANGES_REQUESTED** [review]
Last three points to be clarified, then good to go:
- Failure Reset Time - Could you add information whether "Is this after the first failed attempt or after the last failed attempt?"
- Are login failures be counted while the account is locked?
- Does unlocking account in admin console reset login failures?

File: server_admin/topics/threat/brute-force.adoc [hunk 3]
```diff
@@ -10,13 +10,52 @@ go to the `Brute Force Detection` sub-tab.
...
 
-The way this works is that if there are `Max Login Failures` during a period of `Failure Reset Time`,
-the account is temporarily disabled for the `Wait Increment` multiplied by the number of failures over the max.  After
-`Failure Reset Time` is reached all failures are wiped clean.  The `Max Wait` is the maximum amount of time
-an account can be disabled.  Another preventive measure is that if there are subsequent login failures for one
-account that are too quick for a human to initiate the account will be disabled.  This is controlled by the
-`Quick Login Check Milli Seconds` value.  So, if there are two login failures for the same account within that value,
-the account will be disabled for `Minimum Quick Login Wait`.
+There are 2 different configurations for brute force detection; permanent lockout and temporary lockout. Permanent lockout will disable a users account after an attack is detected; the account will be disabled until an administrator renables it. Temporary lockout will disable a users account for a time period after an attack is detected; the time period for which the account is disabled increases the longer the attack continues.
```
Comment (matthewhelmke) [inline server_admin/topics/threat/brute-force.adoc]: Both instances of "users" should be "user's", with an apostrophe.

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
This is pretty clean, well done! One minor edit is needed.

**Review by matthewhelmke — APPROVED** [review]

**Review by hmlnarik — APPROVED** [review]
Thanks @douglaspalmer!

**Pull request merged**